pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -499,6 +504,7 @@ type ClusterStatus struct {
 	// +optional
 	Address ClusterAddress `json:"address,omitempty"`
 
+	// LastUpdated contains the timestamp at which the cluster was last modified.
```

Comment:
Please mention that this field is deprecated and not being set by KKP anymore. We only kept it in KKP 2.20 to not break backwards-compatibility.